docs/docker.md

```diff
@@ -0,0 +1,40 @@
+# Running agagd locally via docker
+
+## Overview
+
+Docker can automate much of the configuration for you.
+
+
+## Using docker
+
+### Prerequisites
+
+* Recent versions of `docker` and `docker-compose`.
+* A database dump of the full AGAGD schema, stored at `schema.sql`. (in the root directory of the repository)
+
+### Running a dev environment using Docker
+
+Create the file `config-docker.env` (in the repository root) and populate it with the following values:
+
+~~~
+MYSQL_PASSWORD=some-password
+MYSQL_ROOT_PASSWORD=some-other-password
+~~~
```

Comment:
Do you think it would be better if we included this as an an example .env` file for devs to copy with a set of environment vars?

Reply:
Sounds good to me, I'll update the docs and add that.